### PR TITLE
Bump nteract-on-jupyter version

### PR DESCRIPTION
**Pull request by yuvipanda**

### CODE REVIEW

**Comment by yuvipanda** [comment]
@minrk it looks like the 'pip' parts of the frozen environment.yml files are being removed :( Any idea if this is a conda regression or something in our freeze.py?

**Comment by rgbkrk** [comment]
I'm definitely seeing that `pip` section not be included. I'm happy to have nteract on jupyter also on conda forge, I've never looked into that though.

**Comment by minrk** [comment]
pip 18 apparently changed something about how it handles format args. Conda 4.5.11 [adds support](https://github.com/conda/conda/pull/7627) for the pip 18 changes, so rebasing on #391 (and fixing freeze to run with miniconda 4.5.11) gets the pip stuff back.

**Comment by yuvipanda** [comment]
@minrk ah, thanks! Unfortunately, it looks like there's no docker image for miniconda 4.5.11 :( https://hub.docker.com/r/continuumio/miniconda3/tags/. I'll upgrade conda manually inside.

**Comment by yuvipanda** [comment]
I see there was already a commented out line that did this. yay :)

**Comment by yuvipanda** [comment]
Thanks for the conda version tip, @minrk. Looks like that worked.

**Comment by yuvipanda** [comment]
hmm, looks like freeze failed on one of the python versions. re-running to fix.

File: repo2docker/buildpacks/conda/environment.py-3.5.frozen.yml [hunk 1]
```diff
@@ -1,79 +1,9 @@
 # AUTO GENERATED FROM environment.py-3.5.yml, DO NOT MANUALLY MODIFY
-# Frozen on 2018-07-16 23:05:45 UTC
+# Frozen on 2018-09-07 15:52:23 UTC
 name: r2d
 channels:
   - conda-forge
   - defaults
   - conda-forge/label/broken
-dependencies:
```
Comment (betatim) [inline repo2docker/buildpacks/conda/environment.py-3.5.frozen.yml]: No more dependencies at all for PYthon 3.5?

Comment (yuvipanda) [reply repo2docker/buildpacks/conda/environment.py-3.5.frozen.yml]: @betatim yeah, was a bug in freeze.py. Fixed.

**Comment by yuvipanda** [comment]
@rgbkrk gimme a ping if you do!

**Pull request merged**